### PR TITLE
Python3.9 envs

### PR DESCRIPTION
**Pull request by jpa198**
#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

**Comment by jpa198** [comment]
Wiki page with how to perform the upgrade here: https://wiki.canterbury.ac.nz/display/QuakeCore/April+%2723+environment+upgrade+process

**Review by joelridden — APPROVED** [review]

**Comment by sungeunbae** [comment]
chmod +x to update_env_to_py39.sh or update the doc to start with "bash"

**Review by sungeunbae — APPROVED** [review]

**Review by joelridden — APPROVED** [review]
Only thought of maybe adding some comments to the edit_cfg script so it's known the format you are reading / writing. But I think it's simple enough to understand if you actually read the code.

**Review by joelridden — APPROVED** [review]

**Review by joelridden — APPROVED** [review]

File: workflow/environments/org/nesi/update_env_to_py39.sh [hunk 1]
```diff
@@ -8,25 +8,33 @@ export myenv=$envs/$env_name
 if [[ `hostname` =~ 'mahuika' ]] || [[ $HOSTNAME =~ 'wb' ]];then
     module load Python/3.9.9-gimkl-2020a
 
+    python $myenv/workflow/environments/org/nesi/edit_cfg.py $myenv/virt_envs/python3_mahuika/pyvenv.cfg include-system-site-packages true
```
Comment (sungeunbae) [inline workflow/environments/org/nesi/update_env_to_py39.sh:11]: shouldn't this be $myenv/workflow/workflow/environments ?

File: workflow/environments/org/nesi/update_env_to_py39.sh [hunk 2]
```diff
@@ -8,25 +8,33 @@ export myenv=$envs/$env_name
...
     sed -i '1iexport PYTHON_VER=3.9' ~/.bashrc
 
+    python3 -m venv --upgrade $myenv/virt_envs/python3_mahuika
+
     source $myenv/virt_envs/python3_mahuika/bin/activate
 
 elif [[ `hostname` =~ 'maui' ]] || [[ $HOSTNAME =~ 'ni' ]];then
     module load cray-python
 
+    python $myenv/workflow/environments/org/nesi/edit_cfg.py $myenv/virt_envs/python3_maui/pyvenv.cfg include-system-site-packages true
```
Comment (sungeunbae) [inline workflow/environments/org/nesi/update_env_to_py39.sh:27]: Same as above?

**Review by sungeunbae — CHANGES_REQUESTED** [review]

**Review by sungeunbae — APPROVED** [review]

**Pull request merged**